sentences/fr/homeassistant_HassTimerStatus.yaml

```
@@ -6,56 +6,62 @@ intents:
       # No name
       - sentences:
           # Combien de temps reste-t-il
-          - "Combien de temps <reste_t_il>"
+          - "[Encore] Combien de temps <reste_t_il>"
           # Combien de temps reste-t-il au minuteur
-          - "Combien de temps <reste_t_il> au <minuteur>"
+          - "[Encore] Combien de temps <reste_t_il> au <minuteur>"
           # Combien de temps reste-t-il sur le minuteur
-          - "Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur>"
+          - "[Encore] Combien de temps <reste_t_il> (sur|dans) [(le|mon)] <minuteur>"
           # Il reste combien de temps
-          - "Il reste combien [de temps]"
+          - "[Il reste] [encore] combien [de temps]"
```

AH! @bors-ltd Can you make **de temps** mandatory here?

```suggestion
          - "[Il reste] [encore] combien de temps"
```

That's the only one that can yeild "Combien"

---

Oops, that one slipped through. I could also as well split in two sentences:
- [Il reste] [encore] combien de temps
- Il reste [encore] combien

But I don't have a strong opinion, and comply to any guideline you have.